### ISSUE TITLE
Database model planning and refinement

### ISSUE THREAD

**Issue by auishik** [auishik/nsu-ed#4]
@d34thgod  @asimuzzaman  title says it all. It is time to discuss the database model. Heads up! Before you comment check out some online mind mapping or some type of online tool to model our database. [One from me](https://www.mindmeister.com/)

**Comment by auishik** [comment]
[Check this one out!](https://www.vertabelo.com/) It ranked top for most of the searches and possibly the only one to support collaboration.

**Comment by d34thgod** [comment]
sign up krbo naki

**Comment by auishik** [comment]
If this is the only one then yes :smile:

**Comment by auishik** [comment]
Ok seems like this is the only one out there that is free and good and also supports collaboration!. I signed up for the free trial and tinkered around. Everything looks good but there are some caveats. **No real-time collaboration** - so if you start editing then others cannot edit and can only read. The screen updates after every autosave. It is a pain unlike google docs but coupled with skype, we can get the job done.(If we could get the job done here then we do not need another meeting after the holidays) MySql 5.5 is supported and you could even export sql from the model after the design. Summary :- not many options floating around, so we might have to use vertabelo+skype to model the database. Now about the account, **Do not sign up through the normal sign up page!!**

#### Use the academic sign up page

[Here is the academic sign up page](https://my.vertabelo.com/create-new-account/academic) and provide you **univeristy email**. This will enable your academic plan immediately or as they said in the docs.

**Comment by asimuzzaman** [comment]
@auishik , my **Vertabelo** account will be identified by this email address- **asimuzzaman.md@northsouth.edu**

**Comment by auishik** [comment]
@asimuzzaman did you make a project group? If not I will make one and add you! - I will sign up for  the account today and start work from tomorrow. @d34thgod make sure you have the educational account and not the 7 day trial. I will bring my laptop tomorrow and maybe we could finish some work.

**Comment by auishik** [comment]
Heads up! we will also start mapping our code strategy :smile: in something like UML I guess. **_Why?**_ I will be more or less free cause I will be learning AJAX so planning to design the UML diagrams and submit it with our project! @d34thgod @asimuzzaman what do you think?

**Comment by asimuzzaman** [comment]
Yeah we can do UML diagram. First of all we should design the database. @auishik if you haven't already made any project group then I can make one. CSE253 mid made me tired and  took away the time, sorry for that. :disappointed:

**Comment by auishik** [comment]
@asimuzzaman no problem :smile: .
